src/plugins/shell/shell.c

```diff
@@ -70,15 +70,16 @@ int elektraShellGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned, Key * pa
 		int retVal = executeCommand (keyString (cmdKey));
 		if (retVal == -1)
 		{
-			ELEKTRA_SET_ERRORF (144, parentKey, "launching childprocess failed with %s\n", strerror (errno));
+			ELEKTRA_SET_PLUGIN_MISBEHAVIOR_ERRORF (parentKey, "Launching childprocess failed. Reason: %s", strerror (errno));
```

Comment:
I think these are resource errors (forking process usually fails because of too little resources).